dashboard/app/models/user.rb

```diff
@@ -147,9 +143,6 @@ class User < ApplicationRecord
     gender_student_input
     gender_teacher_input
     gender_third_party_input
-    child_account_compliance_state
-    child_account_compliance_state_last_updated
-    child_account_compliance_lock_out_date
```

Comment:
I'm pretty sure there's documentation about these deleted columns. Could you link to the PR that added these properties?

Reply:
https://github.com/code-dot-org/code-dot-org/pull/52244

Reply:
@daynew Do you happen to know anything about the documentation for these fields?

Reply:
Nothing besides what is in the comments of this code.